client/remoterepo.c

```diff
@@ -62,20 +51,36 @@ progress_cb(
     curl_off_t ulNow
     )
 {
-    double dPercent;
+    uint32_t dPercent;
+    pcb_data *pData = (pcb_data *)pUserData;
 
     UNUSED(ulNow);
     UNUSED(ulTotal);
 
-    if (dlTotal <= 0) {
+    if (dlTotal <= 0)
+    {
         return 0;
     }
 
-    dPercent = ((double)dlNow / (double)dlTotal) * 100.0;
-    if (!isatty(STDOUT_FILENO)) {
-        pr_info("%s %3.0f%% %ld\n", (char *)pUserData, dPercent, dlNow);
-    } else {
-        pr_info("%-35s %10ld %5.0f%%\r", (char *)pUserData, dlNow, dPercent);
```

Comment:
Why did you remove the display of `dlNow`? I don't think it's useless information.